examples/calculator.js

```diff
@@ -16,9 +16,33 @@
 
 const assert = require('assert');
 const bacardi = require('bindings')('bacardi.node');
-const calculator = new bacardi.Calculator();
+const calculator = new bacardi.Calculator(0);
+const chai = require('chai');
 const ternary = new bacardi.Ternary();
 
+var expect = chai.expect;
+
+describe('Test for constructor function', () => {
+  describe('there is constructor calculator(long createTime)', () => {
+    it('should be create object without error', () => {
+      new bacardi.Calculator(1000);
+    });
+  });
+  describe('there is no constructor calculator()', () => {
+    it('should be failed create object with error', () => {
+      var badFn = () => {
+        new bacardi.Calculator();
+      };
+      expect(badFn).to.throw(TypeError);
```

Comment:
I thought that we could do it in Mocha as follows:
```Javascript
assert.throws(() => {
     new bacardi.Calculator();
   }, TypeError);
```
If I'm missing something, please correct me :)

Reply:
Wow.. looks so simple! I will try this one.

Reply:
@romandev great! I changed it like you mention.
Thanks your review!